codegen/templates/dependency_struct.tmpl

```diff
@@ -1,5 +1,5 @@
 {{$instance := . -}}
-package {{$instance.PackageInfo.PackageName}}
+package module
```

Comment:
It is solving the cycle issue right now, so I'm keeping it in the module subpackage